CountryPicker/CountryPicker/CountryPickerWithSectionViewController.swift

```diff
@@ -49,6 +49,13 @@ open class CountryPickerWithSectionViewController: CountryPickerController {
         controller.callBack = handler
         
         let navigationController = UINavigationController(rootViewController: controller)
+        
+        if #available(iOS 13.0, *) {
+            
+        } else {
+            navigationController.modalPresentationStyle = .overFullScreen
+        }
+        
```

Comment:
can we do one line 

        if #available(iOS 13.0, *) {  } else {
              navigationController.modalPresentationStyle = .overFullScreen
          }

Reply:
Hello. 
Regarding the screenshot, i am afraid there is no visualization but rather a behavior of the VC. Presenting the CountryPickerWithSectionViewController VC on top of a VC will result in a reloadDate/refresh of the first VC when user selected a country and CountryPickerWithSectionViewController is dismissed. 

hopefully this helps. and do not hesitate if you had any questions.